sh/health.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# shellcheck shell=dash
+set -eo pipefail
+
+CHECK="mariadb"
+
+# prefer root if available
+if [ -n "${MYSQL_ROOT_PASSWORD}" ]; then
```

Comment:
This ENV is not reliable since it can be changed right after the first warm up

Reply:
See my first point/todo and the first comment. MariaDB health check ignores auth and mysql ignores healthchecks altogether.

Reply:
I think what we want is a sane default, if people do changes they should probably override the healthcheck as well.